runtime/vm/classsupport.c

```diff
@@ -167,6 +167,7 @@ findPrimitiveArrayClass(J9JavaVM* vm, jchar sigChar)
 	switch (sigChar) {
 	case 'B':
 		return vm->byteArrayClass;
+		break;
```

Comment:
Would have made more sense to remove the other useless breaks.

Reply:
Done